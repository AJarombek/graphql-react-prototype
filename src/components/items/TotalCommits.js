/**
 * TotalCommits component which displays repositories with the most total commits.
 * @author Andrew Jarombek
 * @since 4/21/2020
 */

import React, { useEffect, useState } from 'react';
import { getTotalCommits } from '../../datasource/GraphQL';

const TotalCommits = () => {
  return (
    <div className="items total-commits">

    </div>
  );
};

export default TotalCommits;
