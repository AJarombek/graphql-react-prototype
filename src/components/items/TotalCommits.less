/**
 * Stylesheet for the TotalCommits component.
 * Author: Andrew Jarombek
 * Date: 4/21/2020
 */

@import "../../vars";

.total-commits {
  grid-area: totCommit;

  h2 {
    margin-bottom: 20px;
  }

  .commits:nth-child(2) {
    p:nth-child(1) {
      font-size: 20px;
    }

    p:nth-child(2) {
      font-size: 20px;
      .hoverPurple();
    }
  }

  .commits {
    display: flex;
    align-items: center;
    margin: 8px 0;
    width: 300px;

    p:nth-child(1) {
      font-size: 16px;
    }

    p:nth-child(2) {
      font-size: 18px;
      margin: 0 0 0 auto;
    }
  }
}
