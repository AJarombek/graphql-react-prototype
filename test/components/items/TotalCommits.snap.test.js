/**
 * Snapshot test for the TotalCommits component.
 * @author Andrew Jarombek
 * @since 4/25/2020
 */

import React from 'react';
import TotalCommits from '../../../src/components/items/TotalCommits';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(<TotalCommits />).toJSON();
  expect(tree).toMatchSnapshot();
});
