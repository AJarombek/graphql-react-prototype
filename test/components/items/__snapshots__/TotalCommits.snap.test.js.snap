// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="items total-commits"
>
  <h2>
    Most Total Commits
  </h2>
</div>
`;
